SETUP.md

```diff
@@ -29,30 +29,13 @@ Follow these steps to get everything that you need installed to build the celo-m
 
 ### Common stuff
 
-#### Install Go
-
-We need Go for [celo-blockchain], the Go Celo implementation, and `gobind` to build Java language bindings to Go code for the Android Geth client).
-
-Note: We currently use Go 1.13. [Install Go 1.13 manually](https://golang.org/dl/), then run
-
-```
-go get golang.org/x/mobile/cmd/gobind
-```
-
-Execute the following (and make sure the lines are in your `~/.bash_profile`):
-
-```
-export GOPATH=$HOME/go
-export PATH=$PATH:$GOPATH/bin
-```
-
```

Comment:
Hi @palango I realised I was mistaken in suggesting that the monorepo doesn't use go, it does for some of the end to end tests that it runs, because it builds celo-blockchain. Could you reinstate this section but just link to the celo-blockchain readme - https://github.com/celo-org/celo-blockchain#building-the-source

Reply:
Thanks for looking at this @piersy , I pushed the changes.